apps/condo/domains/billing/schema/BillingIntegration.js

```diff
@@ -58,6 +62,23 @@ const BillingIntegration = new GQLListSchema('BillingIntegration', {
             isRequired: false,
         },
 
+        group: {
+            adminDoc: 'Any number of billings can have the same billing group. Validations: Should be a sequence of lowercase latin characters.',
+            schemaDoc: 'Billing group which this billing is part of. Used to restrict certain billings from certain acquirings"',
+            type: Text,
+            isRequired: true,
+            defaultValue: DEFAULT_BILLING_INTEGRATION_GROUP,
+            hooks: {
+                validateInput: async ({ resolvedData, addFieldValidationError }) => {
```

Comment:
`resolvedData[fieldPath]` can be undefined on update if it's not in payload....
```js
if ('group' in resolvedData) {
// logic
}
```

Reply:
Do we have a test for this case? Wonder, how its passed

Reply:
Yep, basically its any update case

It passed because apparently regex test is satisfied with `undefined` :-) added a verbose check: (if operation = update, then do not check)